### DISCUSSION TITLE
部署到GP，并使用自定义域名，每一次自动部署后都需要更新自定义域名

### DISCUSSION THREAD

**Discussion by le0zh0u** [vuepress/core#1023] [Q&A] [answered]
参照[部署-Github Page](https://v2.vuepress.vuejs.org/zh/guide/deployment.html)的文档，配置了workflow。

但因为使用自定义域名会出现一个问题是，每一次push之后，自动出发了部署。

因为Build之后的目录中不包含CNAME，导致之前设置的自定义域名失效。

大家有碰到类似的问题吗？

**Comment by Mister-Hope** [comment, marked as answer]
你不会自动在部署前生成一个么？再或者直接在oublic文件夹弄一个

**Reply by le0zh0u** [reply]
感谢，按照现在的部署逻辑，是build完成后将dist下的文件放到gh-page的分支下面。后来解决了，增加了一个action，在完成build后，在dist目录下生成CNAME，这样自己部署后会自动使用自定义域名。

之前提出，以为是通用的能力 \捂脸
